src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/ActionButtons.css

```diff
@@ -0,0 +1,7 @@
+.discovery-modal_buttons-row button {
+  height: auto;
+  min-height: 54px;
```

Comment:
This seems to be too high, I think it would be better if the button height can fill parent container height

Reply:
Good idea. Removed min-height, set height to 100% and set line-height to 1.2 to decrease overall button height while retaining a consistent height across all buttons. 

Design: 
<img width="586" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/437b36a9-341b-4217-a4ab-d0e7b9bcb9ca">

Before: 
<img width="343" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/283c71eb-dc2b-484b-aa5e-be5117f5686c">

After: 
<img width="719" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/3828a4c4-0d91-4450-9e8d-7184667e22d3">